### PR TITLE
Fixes the logo height

### LINKED ISSUE #11372: Top-left logo reaches 1 px beyond the header
The logo in the top-left corner has a absolute position of 1 px from top. With the 100% height it has the same heigth as the header and thus is shown 1 px beyond the header, if the logo has no space around it.

Example
I used the browser to zoom in as 1 px is hard to see. ;-)

### PR DESCRIPTION
**Pull request by weeman1337**
before
image

after
image

Closes #11372

### CODE REVIEW

File: core/css/header.scss [hunk 1]
```diff
@@ -113,7 +113,7 @@
 		background-size: contain;
 		background-position: center;
 		width: 62px;
-		height: 100%;
+		height: calc(100% - 2px);
```
Comment (skjnldsv) [inline core/css/header.scss]: I'm cinfused, where does those 2px come from? :thinking: :grin:

Comment (weeman1337) [reply core/css/header.scss]: @skjnldsv it has `top: 1px;`. When there should be the same at the bottom a height of 100 % minus two pixels would to that.

Comment (skjnldsv) [reply core/css/header.scss]: Shouldn't we set a top at 0?
I remember something about this, but I can't recall :grin:

Comment (skjnldsv) [reply core/css/header.scss]: Found the guilty man: https://github.com/nextcloud/server/pull/10853
Damn you @skjnldsv!!!!

Comment (weeman1337) [reply core/css/header.scss]: ha ha :) I tried it without the tiny spacing. In my opinion it looks better like this...

Comment (skjnldsv) [reply core/css/header.scss]: you could add a bottom: 1px? so we understand better?
I'm fine with that, but i'm afraid we'll forget about this like I did above :stuck_out_tongue_closed_eyes:

Comment (weeman1337) [reply core/css/header.scss]: @skjnldsv done ✓

**Review by skjnldsv — APPROVED** [review]

**Comment by go2sh** [comment]
I tested it also. :+1:

**Comment by skjnldsv** [comment]
@go2sh you should be able to add your review on the top right ;)

**Review by go2sh — APPROVED** [review]
Tested and works. :)

**Review by rullzer — APPROVED** [review]

**Comment by MorrisJobke** [comment]
@weeman1337 Once the backport PR is open you could remove the `backport-request` label. Because we go regularly through the closed PRs that still have the label to backport those. This then makes it easier for us to spot the not yet backported. Just a little hint for the future and to explain how we work. :)

**Pull request merged**